src/AuthenticatedApp.tsx

```diff
@@ -59,13 +59,20 @@ const Wrapper: FunctionComponent<RouteComponentProps & {sessionData: UserSession
 
 const AuthenticatedApp: FunctionComponent<RouteComponentProps> = ({location, match}) => {
   const sessionData = useUserSessionDataState()
+  const sessionUpdateFn = useUserSessionDataDispatch()
 
   if (!sessionData.token) {
```

Comment:
Is sessionData.token ever set?  Why is session storage used at all (couldn't it just rely on the in-memory context, and send the user to login if not set)?
I'm not recommending a rewrite as part of this issue/PR, but these are questions that the Bridge team may want to follow-up on in the future.

Reply:
`sessionData.token` is only set in React context -- when sessionData is saved to sessionStorage in [`AuthContext`](https://github.com/hallieswan/mtb/blob/91bc7f54d417629cca16b393e4205bad061bde1f/src/helpers/AuthContext.tsx#L65), the token is stored as `undefined`. I agree that we could rely on the in-memory context for the auth token and send the user to login if not set. sessionStorage is used by two methods (`isInAdminRole`, `isPathAllowed`) to get the user's `AdminRoles`, but we could refactor. I opened [a ticket](https://sagebionetworks.jira.com/browse/DHP-1083) to track the work for future follow-up.